src/pipert2/core/base/logger.py

```diff
@@ -2,15 +2,67 @@
 
 
 class PipeLogger(ABC):
```

Comment:
why don't use the regular python logging library?

Reply:
We will, but its an interface if we want to use other types of loggers, for example splunk.
We implement this interface so that no matter what type of logger the user will decide to use (python logging or splunk), the pipe will have a logger with known methods no matter its type.